app/src/main/kotlin/net/sourceforge/opencamera/uploadservice/FileUploadService.kt

```diff
@@ -0,0 +1,51 @@
+package net.sourceforge.opencamera.uploadservice
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.os.Build
+import net.gotev.uploadservice.protocols.multipart.MultipartUploadRequest
+import net.sourceforge.opencamera.MainActivity
+import java.io.File
+
+class FileUploadService(private val mMainActivity: MainActivity) {
+
+    /**
+     * Create channel that notifies the user of file upload in progress.
+     */
+    fun createUploadServiceNotificationChannel() {
+        if (Build.VERSION.SDK_INT >= 26) {
```

Comment:
I think that this method should be called only for API 26, because of his naming --- also `if` wrapping the whole method isn't a great idea

Reply:
ok